### ISSUE TITLE
ksniff fails when tar is not installed in container

### ISSUE THREAD

**Issue by ottoyiu** [eldadru/ksniff#25]
ksniff fails when running tar if it does not exist. This is the verbose logs when running on an image that has a statically compiled binary in a Docker 'scratch' image.

```
INFO[0000] running in verbose mode
DEBU[0000] searching for tcpdump binary using lookup list: '[ static-tcpdump /usr/local/bin/static-tcpdump /home/oyiu/.kube/plugin/sniff/static-tcpdump]'
DEBU[0000] tcpdump binary was not found at: ''
DEBU[0000] tcpdump binary was not found at: 'static-tcpdump'
DEBU[0000] tcpdump binary found at: '/usr/local/bin/static-tcpdump'
INFO[0000] using tcpdump path at: '/usr/local/bin/static-tcpdump'
DEBU[0000] pod 'goldpinger-8kwlm' status: 'Running'
INFO[0000] no container specified, taking first container we found in pod.
INFO[0000] selected container: 'goldpinger'
INFO[0000] sniffing on pod: 'goldpinger-8kwlm' [namespace: 'oyiu-test', container: 'goldpinger', filter: '', interface: 'eth0']
INFO[0000] checking for static tcpdump binary on: '/tmp/static-tcpdump'
DEBU[0000] checked for tcpdump on remote pod: exit-code: '126', stdout: 'OCI runtime exec failed: exec failed: container_linux.go:348: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory": unknown
', stderr: ''
INFO[0000] couldn't find static tcpdump binary on: '/tmp/static-tcpdump', starting to upload
DEBU[0000] uploading file from: '/usr/local/bin/static-tcpdump' to '/tmp/static-tcpdump'
DEBU[0000] read '/usr/local/bin/static-tcpdump' to memory, file size: '2642872'
DEBU[0000] formatted '/usr/local/bin/static-tcpdump' as tar, tar size: '2644480'
DEBU[0000] executing tar: '[tar -xf - -C /tmp]'
DEBU[0001] done uploading file, exitCode: '126', stdOut: 'OCI runtime exec failed: exec failed: container_linux.go:348: starting container process caused "exec: \"tar\": executable file not found in $PATH": unknown
', stdErr: ''
INFO[0001] spawning wireshark!
ERROR: logging before flag.Parse: E0111 17:03:52.436667   10386 v2.go:105] write tcp 10.49.16.74:52046->10.49.252.10:443: use of closed network connection
DEBU[0001] executing tcpdump on remote pod
DEBU[0001] tcpdump executed, exitCode: '126', stdErr: '&{}'  error="<nil>"
^C
```

**Comment by eldadru** [comment]
It's sort of a known issue, ksniff (and kubectl as well) use tar to upload files to the container, if it's missing than the upload fails.

I do agree that ksniff should first verify that the remote container has "tar" before trying to upload.
In the future we should have support for containers without "tar" - such as scratch containers.
We might do that as part of #20 I'm still thinking about that.

So the action item here is to add verification that tar exists.
Thanks!

**Comment by eldadru** [comment]
OK, this issue will be addressed as part of #20 which will also support scratch containers :)
